cmd/mayactl/app/command/volume_stats_test.go

```diff
@@ -473,24 +595,46 @@ func TestDisplayStats(t *testing.T) {
 				UsedLogicalBlocks:    "1048576",
 				WriteIOPS:            "20",
 			},
+			volume: v1alpha1.CASVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						"vsm.openebs.io/controller-ips":    "10.48.1.17",
```

Comment:
Are all these annotations required. What would happen if some of the annotations are missing like `vsm.xxxx` ? mayactl should be able to handle that. Create a test case for that too.

Reply:
Have handled this in CASVolume test function.